settings.gradle

```diff
@@ -1,9 +1,10 @@
 pluginManagement {
     apply from: 'versions.gradle'
 
+    // Dokka artifacts location
     repositories {
         maven {
-            url "file://opt/dokka/repository"
+            url = "https://software.r3.com/artifactory/conclave-maven"
```

Comment:
Everything looks good. I will approve once we have decided what to do about this repo.